pkg/operator/target_config_reconciler.go

```diff
@@ -1287,3 +1540,43 @@ func (c *TargetConfigReconciler) eventHandler() cache.ResourceEventHandler {
 		DeleteFunc: func(obj interface{}) { c.queue.Add(workQueueKey) },
 	}
 }
+
+func (c *TargetConfigReconciler) checkNamepsaceMonitoringLabel() error {
+	operatorNamespace, err := c.kubeClient.CoreV1().Namespaces().Get(c.ctx, operatorclient.OperatorNamespace, metav1.GetOptions{})
```

Comment:
This needs to be changed to read the namespace from the informer, not directly accessing the apiserver to minimize the traffic.

Reply:
done